server/app/views/LoginForm.java

```diff
@@ -90,22 +94,23 @@ private ContainerTag mainContent(Messages messages) {
             .with(p(civicEntityShortName).withClasses(Styles.FONT_BOLD))
             .with(p("CiviForm")));
 
-    String loginMessage =
-        messages.at(MessageKey.CONTENT_LOGIN_PROMPT.getKeyName(), civicEntityFullName);
-    content.with(
-        div()
-            .withClasses(
-                Styles.FLEX,
-                Styles.FLEX_COL,
-                Styles.GAP_2,
-                Styles.PY_6,
-                Styles.PX_8,
-                Styles.TEXT_LG,
-                Styles.W_FULL,
-                Styles.PLACE_ITEMS_CENTER)
-            .with(p(loginMessage))
-            .with(loginButton(messages)));
-
+    if (applicantIdpName != AuthIdentityProviderName.DISABLED_APPLICANT) {
```

Comment:
can we have a message saying "resident auth is disabled"?

Reply:
Done